aspnetcore/signalr/get-started-signalr-core.md

```diff
@@ -0,0 +1,107 @@
+* [Visual Studio 2017](https://www.visualstudio.com/downloads/) version 15.6 or later with the ASP.NET and web development workload.
+
+## Create an ASP.NET Core project that uses SignalR
+
+1. Use the **File** > **New Project** menu option and choose **ASP.NET Core Web Application**.
+
+  ![New Project dialog in Visual Studio](get-started-signalr-core/_static/signalr-new-project-dialog.png)
+
+1. In the next step, choose **Web Application** to use a lightweight project template for using Razor Pages. Then select **Ok**.
+
+  ![New Project dialog in Visual Studio](get-started-signalr-core/_static/signalr-new-project-choose-type.png)
+
+1. Add the `Microsoft.AspNetCore.SignalR` NuGet package to the ASP.NET Core project. You can do this by selecting **Tools** > **NuGet Package Manager** > **Package Manager Console**. In the console, issue  the following command:
+
+  ```text
+  Install-Package Microsoft.AspNetCore.SignalR -Version 1.0.0-alpha2-final
+  ```
+
+  This command installs the `Microsoft.AspNetCore.SignalR` NuGet package that contains libraries SignalR server-side code.
+
+1. Install the JavaScript client library by using npm. `npm install` creates a *\node_modules\* folder and sub-folders for the libraries in the same location as where you run the npm command.
+
+  ```
+   npm install @aspnet/signalr-client
+  ```
+
+1. Copy the *signalr-client-1.0.0-alpha2-final.js* for SignalR from *\node_modules\@aspnet\signalr-client\dist\browser* to the *wwwroot\lib* folder in your project.
+
+## Create the SignalR Hub
+
+A hub is a class that represents a high-level pipeline that allows the client and server to call methods on each other directly.
+
+1. Add a class to the project by choosing **File** > **New** > **File** and selecting **Visual C# Class**. 
+
+1. Inherit from `Microsoft.AspNetCore.SignalR.Hub`. The `Hub` class contains properties and events for managing connections and groups, as well as sending and receiving data.
+
+1. Create the `Send` method that sends a message to all connected chat clients. Notice it returns a `Task`, because SignalR is asynchronous and scales better.
+
+  [!code-csharp[Startup](get-started-signalr-core/sample/ChatHub.cs?range=7-14)]
+
+## Configure the project to use SignalR
+
+The SignalR server must be configured so that it knows to pass requests to SignalR.
+
+1. To configure a SignalR project, modify the `ConfigureServices` method of the application's `Startup` class by inserting a call to `services.AddSignalR()`.
+
+  This adds SignalR as part of the [ASP.NET Core middleware](xref:fundamentals/middleware/index) pipeline.
+
+1. In the `Startup` class you must also configure SignalR routes by calling `UseSignalR`. In the call to `UseSignalR` is where you setup the SignalR hub mappings.
+
+  [!code-csharp[Startup](get-started-signalr-core/sample/Startup.cs?highlight=22,40-43)]
+
+## Create the SignalR client code
+
+1. Open and replace the content in `\Pages\Index.cshtml` with the following code:
```

Comment:
reformat markup so there is no horiz scroll bar using the widest window with no left or right chrome
![image](https://user-images.githubusercontent.com/3605364/36638400-10c6cd9c-1998-11e8-9a77-7f90aa894e0a.png)